packages/tss-lib/webpack.config.js

```diff
@@ -12,5 +12,5 @@ const cfg = {
 
 module.exports = {
   umdConfig: cfg,
-  cjsConfig: cfg,
+  cjsConfig: cfg
```

Comment:
there is a config for cjs but `libCjs: false` in `torus.config.js`?

Reply:
replied above